docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -834,20 +856,20 @@ public class CustomTenantResolver implements TenantResolver {
 ----
 
 [[disable-tenant]]
-== Disabling Tenant Configurations
+== Disabling tenant configurations
 
-Custom `TenantResolver` and `TenantConfigResolver` implementations may return `null` if no tenant can be inferred from the current request and a fallback to the default tenant configuration is required.
+Custom `TenantResolver` and `TenantConfigResolver` implementations might return `null` if no tenant can be inferred from the current request and a fallback to the default tenant configuration is required.
 
-If you expect that the custom resolvers will always infer a tenant then you do not need to configure the default tenant resolution.
+If you expect the custom resolvers to always infer a tenant, you do not need to configure the default tenant resolution.
```

Comment:
```suggestion
If you expect the custom resolvers always to infer a tenant, you do not need to configure the default tenant resolution.
```

Reply:
This looks fine, but I've noticed `infer` was introduced at some point, I had to do a quick Google search to make sure I know what this word means. Propose to replace it with `resolve`, there is some concrete check as opposed to some deduction logic here, `resolve` is a typical term in this context

Reply:
Updated to use "resolve" rather than "infer". Preserving the current word order, "to always".